docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - cinema-media:/files/media
+    command: >
+      sh -c "python manage.py migrate &&
+              python manage.py runserver 0.0.0.0:8000"

```

Comment:
Consider using a separate entrypoint script for running multiple commands. This can improve readability and maintainability of the Docker setup.